### PR TITLE
s5t4-addInputToDb 

### PR DESCRIPTION
**Pull request by nesclark**
we've updated db with new field methods to addCourse and bindParams

### CODE REVIEW

File: src/Models/CourseModel.php [hunk 1]
```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
...
+            :in_person_input,
+            :remote_input);");
 
         $startDate = $newCourse['startDate'];
         $endDate = $newCourse['endDate'];
         $name = $newCourse['courseName'];
         $notes = $newCourse['notes'];
         $in_person = $newCourse['in_person'];
         $remote = $newCourse['remote'];
+        $in_person_input = $newCourse['in_person_input'];
```
Comment (Steph-ski) [inline src/Models/CourseModel.php:69]: should this be $in_person_spaces_input?

File: src/Models/CourseModel.php [hunk 2]
```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
...
+        $remote_input = $newCourse['remote_input'];
```
Comment (Steph-ski) [inline src/Models/CourseModel.php:70]: should this be $remote_spaces_input?

File: public/js/addCourse.js [hunk 3]
```diff
@@ -52,6 +52,7 @@ courseForm.addEventListener('submit', e => {
         }
     });
     if (formIsValid) {
+        console.log(data)
```
Comment (Steph-ski) [inline public/js/addCourse.js:55]: remove this?

File: src/Controllers/API/AddCourseController.php [hunk 4]
```diff
@@ -35,6 +35,7 @@ public function __invoke(Request $request, Response $response, array $args): Res
         try {
             if (CourseValidator::validate($newCourse)) {
                 $newCourse = CourseSanitiser::sanitise($newCourse);
+                var_dump($newCourse);
```
Comment (Steph-ski) [inline src/Controllers/API/AddCourseController.php:38]: remove this?

**Comment by HadertS** [comment]
db file needs updating with the new fields.

File: src/Models/CourseModel.php [hunk 5]
```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
...
             ) 
             VALUES (
             :startDate, 
             :endDate, 
             :name,
             :notes,
             :in_person,
-            :remote);");
+            :remote,
+            :in_person_input,
```
Comment (Steph-ski) [inline src/Models/CourseModel.php:60]: should this be $in_person_spaces_input?

File: src/Models/CourseModel.php [hunk 6]
```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
...
+        $remote_input = $newCourse['remote_input'];
 
         $query->bindParam(':startDate', $startDate);
         $query->bindParam(':endDate', $endDate);
         $query->bindParam(':name', $name);
         $query->bindParam(':notes', $notes);
         $query->bindParam(':in_person', $in_person);
         $query->bindParam(':remote', $remote);
+        $query->bindParam(':in_person_input', $in_person_input);
+        $query->bindParam(':remote_input', $remote_input);
```
Comment (Steph-ski) [inline src/Models/CourseModel.php:79]: should this be $remote_spaces_input?

File: src/Models/CourseModel.php [hunk 7]
```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
             `name`,
             `notes`,
             `in_person`,
-            `remote`
+            `remote`,
+            `available_spaces_in_person`,
```
Comment (Steph-ski) [inline src/Models/CourseModel.php:50]: should we keep this in line with the naming in the code base? i.e in_person_spaces?

File: src/Models/CourseModel.php [hunk 8]
```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
...
+            `available_spaces_remote`
```
Comment (Steph-ski) [inline src/Models/CourseModel.php:51]: should we keep this in line with the naming in the code base? i.e remote_spaces?

**Review by Steph-ski — CHANGES_REQUESTED** [review]
2 instances of testing code can be removed. 

Also, think we changed the names of the inputs to the following as one of the stories yesterday so it's more semantic? Should we remove the _input all together? 
$in_person_spaces_input
$remote_spaces_input

When I run the form in the browser, there is an error which looks likes it due to not exporting the updated db. Also added a comment about the db column names being more inline with the codebase naming.

File: public/js/addCourse.js
Comment (HadertS) [inline public/js/addCourse.js:1]: Lines 120 and 121 need to be updated to use the new variable names for in_person_spaces / remote_spaces
lines 150 and 155 need to be similarly updated

File: db/academyPortal.sql
Comment (HadertS) [inline db/academyPortal.sql:1]: This needs to be updated to have the new field names - outdated

**Pull request merged**